Unit Tests/Tests/Redux/Reducers/Model/UserReducerTests.swift

```diff
@@ -0,0 +1,79 @@
+import XCTest
+import TestUtilities
+@testable import PusherChatkit
+
+class UserReducerTests: XCTestCase {
+    
+    // MARK: - Tests
+    
+    func test_reduce_withCurrentStateAndInitialStateAction_returnsModifiedState() {
```

Comment:
A lot of your tests include `withCurrentStateAnd` in their name which I don't think adds any value (because there has to be a current state thats passed to every `redude` function.  Perhaps you drop this to make the method names more succinct (like in `MasterReducerTests`)

Reply:
Sure, I can drop this.